english/132-days-ago/132-days-ago.md

```diff
@@ -0,0 +1 @@
+132 days ago you went to Tesco to buy marshmallow but end up buying chocolate because you had not enough money to buy marshmallow.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.